CONTRIBUTING.md

```diff
@@ -16,27 +16,27 @@ $ yarn
 ## Building
 
 ```sh
-$ npm run build
+$ yarn build
```

Comment:
I know Yarn works without specifying `run` before the command, but it's probably safest that way since that is what is documented. Can we change these (e.g. `yarn run build`)?

Reply:
Thanks @tech4him1 . Change it back to yarn run.